packages/tokens/README.md

```diff
@@ -45,6 +45,53 @@ The tokens package ships with two themes: 🌞 light and 🌛 dark.
 </html>
 ```
 
+#### Switching themes during runtime
+
+You are also able to switch themes during the runtime. For the time being, we do not ship a utility function for this, as it is easy to implement. Each theme applies the variables via a `:root` selector, as well as a `className` that may be added to the `document.body`. To switch the theme, proceed in the following way:
+
+```html
+<!DOCTYPE html>
+  <head>
+    <!--
+      -- Load both themes initially.
+      -- The last theme will be the default, in this case the light theme
+    -->
+    <link rel="stylesheet" href="/node_modules/@synergy-design-system/tokens/themes/dark.css" />
+    <link rel="stylesheet" href="/node_modules/@synergy-design-system/tokens/themes/light.css" />
+  </head>
+  <body>
+    <button id="theme-switch">Switch Theme</button>
+    <script>
+    const switchTheme = ({ target }) => {
+      const bdy = document.body;
```

Comment:
See comment for https://github.com/synergy-design-system/synergy-design-system/pull/238/files#diff-ab22fd38a77da3408a668fd2133027984d431c3eef05093a33705bae0e3f87ccR13

Reply:
I had problems initially, but went with descructuring to work around them. Seemed to be a linter issue `const body` would also have been worked.